desktop/docs/secrets.md

```diff
@@ -14,10 +14,11 @@ and click on the create button. The other fields may be left blank.
 In the end, your application details should look similar to:
 ![application details](https://cldup.com/Juw-5uEHlR.png)
 
-After your application is created, copy your client id and client secret, and add a `calypso/config/secrets.json` file:
+After your application is created, copy your client id and client secret, and either add or update `/config/secrets.json` file and be sure to include the following properties:
```

Comment:
I think so too...

I might not fully understand the situation but I wonder if either of these options would make sense?:

- Include the secrets file with empty values for each of the expected keys. I feel that this would avoid having to explain that these keys should be added, only the values which seems more intuitive.

- The secrets file used in the desktop build could live within `/desktop` - perhaps this will make the instructions less confusing? Though I think this could be problematic down the line if other 'secrets' are needed that should live outside of the scope of `/desktop`.

Either way I think the process of adding the secrets could be improved by having something like `make secrets` - A quick CLI for creating the file with the correct keys - any thoughts on this?